build.psm1

```diff
@@ -462,7 +462,10 @@ Fix steps:
     # ARM is cross compiled, so we can't run pwsh to enumerate Experimental Features
```

Comment:
@TravisEz13, your last commit had 6 failures in `PowerShell-CI-windows`
(These are 5 of the failures)

Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:15 char:37
+                 @{ Script = { $true ? 1 : 2 };  ExpectedValue = 1 }
+                                     ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:16 char:38
+                 @{ Script = { $true? ?1 :2 };   ExpectedValue = 2 }
+                                      ~~
Unexpected token '?1' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:17 char:38
+                 @{ Script = { ${true}?1:2 };    ExpectedValue = 1 }
+                                      ~~~~
Unexpected token '?1:2' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:18 char:33
+                 @{ Script = { 1 ? 1kb : 0xf };  ExpectedValue = 1kb }
+                                 ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:19 char:33
+                 @{ Script = { 0 ?1kb:0xf };     ExpectedValue = 15 }
+                                 ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:20 char:35
+                 @{ Script = { 's' ?1kb:0xf };   ExpectedValue = 1kb }
+                                   ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:21 char:37
+                 @{ Script = { $null ?1kb:0xf }; ExpectedValue = 15 }
+                                     ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:22 char:34
+                 @{ Script = { '' ?1kb:0xf };    ExpectedValue = 15 }
+                                  ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:25 char:39
+                 @{ Script = { 1,2,3,4 ? 'Core' : 'Desktop' };         ...
+                                       ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:26 char:42
+                 @{ Script = { @(1,2,3,4) ? 'Core' : 'Desktop' };      ...
+                                          ~
Unexpected token '?' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:15 char:37
+                 @{ Script = { $true ? 1 : 2 };  ExpectedValue = 1 }
+                                     ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:16 char:38
+                 @{ Script = { $true? ?1 :2 };   ExpectedValue = 2 }
+                                      ~~
Unexpected token '?1' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:17 char:38
+                 @{ Script = { ${true}?1:2 };    ExpectedValue = 1 }
+                                      ~~~~
Unexpected token '?1:2' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:18 char:33
+                 @{ Script = { 1 ? 1kb : 0xf };  ExpectedValue = 1kb }
+                                 ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:19 char:33
+                 @{ Script = { 0 ?1kb:0xf };     ExpectedValue = 15 }
+                                 ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:20 char:35
+                 @{ Script = { 's' ?1kb:0xf };   ExpectedValue = 1kb }
+                                   ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:21 char:37
+                 @{ Script = { $null ?1kb:0xf }; ExpectedValue = 15 }
+                                     ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:22 char:34
+                 @{ Script = { '' ?1kb:0xf };    ExpectedValue = 15 }
+                                  ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:25 char:39
+                 @{ Script = { 1,2,3,4 ? 'Core' : 'Desktop' };         ...
+                                       ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:26 char:42
+                 @{ Script = { @(1,2,3,4) ? 'Core' : 'Desktop' };      ...
+                                          ~
Unexpected token '?' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:15 char:37
+                 @{ Script = { $true ? 1 : 2 };  ExpectedValue = 1 }
+                                     ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:16 char:38
+                 @{ Script = { $true? ?1 :2 };   ExpectedValue = 2 }
+                                      ~~
Unexpected token '?1' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:17 char:38
+                 @{ Script = { ${true}?1:2 };    ExpectedValue = 1 }
+                                      ~~~~
Unexpected token '?1:2' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:18 char:33
+                 @{ Script = { 1 ? 1kb : 0xf };  ExpectedValue = 1kb }
+                                 ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:19 char:33
+                 @{ Script = { 0 ?1kb:0xf };     ExpectedValue = 15 }
+                                 ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:20 char:35
+                 @{ Script = { 's' ?1kb:0xf };   ExpectedValue = 1kb }
+                                   ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:21 char:37
+                 @{ Script = { $null ?1kb:0xf }; ExpectedValue = 15 }
+                                     ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:22 char:34
+                 @{ Script = { '' ?1kb:0xf };    ExpectedValue = 15 }
+                                  ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:25 char:39
+                 @{ Script = { 1,2,3,4 ? 'Core' : 'Desktop' };         ...
+                                       ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:26 char:42
+                 @{ Script = { @(1,2,3,4) ? 'Core' : 'Desktop' };      ...
+                                          ~
Unexpected token '?' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```
Default enablement of Experimental Features.On preview builds, Experimental Features are enabled
```powershell
Expected: Feature PSCommandNotFoundSuggestion to be Enabled
at <ScriptBlock>, D:\a\1\s\test\powershell\engine\ExperimentalFeature\Get-ExperimentalFeature.Tests.ps1: line 176
176:             $expFeature.Enabled | Should -BeEnabled -Name $expFeature.Name
```
Error occurred in test script 'D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1'
```powershell
At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:15 char:37
+                 @{ Script = { $true ? 1 : 2 };  ExpectedValue = 1 }
+                                     ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:16 char:38
+                 @{ Script = { $true? ?1 :2 };   ExpectedValue = 2 }
+                                      ~~
Unexpected token '?1' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:17 char:38
+                 @{ Script = { ${true}?1:2 };    ExpectedValue = 1 }
+                                      ~~~~
Unexpected token '?1:2' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:18 char:33
+                 @{ Script = { 1 ? 1kb : 0xf };  ExpectedValue = 1kb }
+                                 ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:19 char:33
+                 @{ Script = { 0 ?1kb:0xf };     ExpectedValue = 15 }
+                                 ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:20 char:35
+                 @{ Script = { 's' ?1kb:0xf };   ExpectedValue = 1kb }
+                                   ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:21 char:37
+                 @{ Script = { $null ?1kb:0xf }; ExpectedValue = 15 }
+                                     ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:22 char:34
+                 @{ Script = { '' ?1kb:0xf };    ExpectedValue = 15 }
+                                  ~~~~~~~~
Unexpected token '?1kb:0xf' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:25 char:39
+                 @{ Script = { 1,2,3,4 ? 'Core' : 'Desktop' };         ...
+                                       ~
Unexpected token '?' in expression or statement.

At D:\a\1\s\test\powershell\Language\Operators\TernaryOperator.Tests.ps1:26 char:42
+                 @{ Script = { @(1,2,3,4) ? 'Core' : 'Desktop' };      ...
+                                          ~
Unexpected token '?' in expression or statement.

Not all parse errors were reported.  Correct the reported errors and try again.
at <ScriptBlock>, D:\a\1\s\src\powershell-win-core\bin\Release\netcoreapp3.0\win7-x64\publish\Modules\Pester\4.8.0\Pester.psm1: line 1097
```